cairo/ethereum/cancun/transactions_types.cairo

```diff
@@ -120,3 +120,173 @@ struct TransactionStruct {
 struct Transaction {
     value: TransactionStruct*,
 }
+
+namespace TransactionType {
+    const LEGACY = 0;
+    const ACCESS_LIST = 1;
+    const FEE_MARKET = 2;
+    const BLOB = 3;
+}
+
+namespace TransactionImpl {
+    func _get_transaction_type(tx: Transaction) -> felt {
```

Comment:
_ is used usually for internal function. 
But this one is also used publicly 
I would remove it